modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ArticleBodyRenderer.es.js

```diff
@@ -24,11 +24,20 @@ export default ({
 	id,
 	signature,
 }) => {
+	const [
+		aricleBodyContainsParagraph,
```

Comment:
here. aricle instead of article

Reply:
🤦‍♂️ 🤦‍♂️ 🤦‍♂️ 
I will change right now. Sorry :(

Reply:
Fixed :)